README.md

```diff
@@ -1,8 +1,10 @@
 # Rack::ContentTypeDefault
 
 Rack::ContentTypeDefault is a tiny piece of middleware that sets default `CONTENT-TYPE:` header when it isn't present
-for specific methods. It checks if a content type is set and does not override the existing one.
-The options allow you to specify the request methods, and the content type to be set.
+for specific requests. It checks if a content type is set and does not override the existing one.
+The options allow you to specify the request methods and the content type to be set. The options also allow you to set
```

Comment:
in fact i would flip 3 & 4. sorry i thought path was the third param. but it's the last.